app/src/main/java/org/mozilla/focus/settings/ManualAddSearchEngineSettingsFragment.java

```diff
@@ -45,6 +49,15 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
+            case R.id.learn_more:
+                final Context context = getActivity();
+                final String url = SupportUtils.getSumoURLForTopic(context, "add-search-engine");
+
+                final String title = ((AppCompatActivity) getActivity()).getSupportActionBar().getTitle().toString();
```

Comment:
Wouldn't it be simpler to just use the String resource instead of trying to get it from the activity?

Reply:
I'm trying to keep the same app bar string as the one that appears in Settings when the user enters this WebView.

With this approach (using the String resource), I'm concerned that we might change the code to point to a different String resource in settings (or move this fragment) and then this code will no longer point to the correct String.